official/resnet/resnet.py

```diff
@@ -454,7 +555,7 @@ def resnet_model_fn(features, labels, mode, model_class,
   # Generate a summary node for the images
   tf.summary.image('images', features, max_outputs=6)
 
-  model = model_class(resnet_size, data_format)
+  model = model_class(resnet_size, data_format, version=version)
```

Comment:
I think the ordering here is inconsistent with the order of args in init? Oh wait, the child classes have data_format and version in a different order than the parent. That seems confusing-- can we standardize the ordering?

Reply:
If memory serves the order got weird because python doesn't allow args without defaults to come after those with. (I think that's the reason I eventually set the version=None default.) I'll see what I can do to make everything cleaner.